dotcom-rendering/src/layouts/GalleryLayout.tsx

```diff
@@ -156,30 +165,9 @@ export const GalleryLayout = (props: WebProps | AppProps) => {
 						/>
 					) : null}
 					<div
-						css={[
-							css`
-								${grid.column.centre}
-								${from.leftCol} {
-									${grid.column.left}
-								}
-								position: relative;
-								${between.tablet.and.leftCol} {
-									&::before {
-										content: '';
-										position: absolute;
-										left: -10px;
-										top: 0;
-										bottom: 0;
-										width: 1px;
-										background-color: ${palette(
-											'--article-border',
-										)};
-									}
-								}
-							`,
-						]}
+						css={[grid.between('centre-column-start', 'grid-end')]}
 					>
-						Main media caption
+						Meta
```

Comment:
Why do we need a placeholder for the meta section? It looks like this section has already been implemented above?